Project.toml

```diff
@@ -17,14 +17,14 @@ LaTeXStrings = "^1.3"
 NaNMath = "0.3, 1"
 Requires = "1"
 Tables = "1.11"
-TikzPictures = "3.5"
 julia = "^1.6"
 
```

Comment:
You also need to remove https://github.com/JuliaSmoothOptimizers/BenchmarkProfiles.jl/blob/e639a63f8513a3da75e2b07297db09aca4113f26/Project.toml#L12